### PR TITLE
Fix tiny typo: Diapora -> Diaspora

### PR DESCRIPTION
**Pull request by mlandauer**

### CODE REVIEW

**Comment by paskalito** [comment]
OMG ^^ Epic - the only one who is visible ^^
thank you :)

File: app/views/frontend/shared/_embedshare.haml [hunk 1]
```diff
@@ -23,7 +23,7 @@
   %li
     %a{:id => "btnDiaspora", :href => diaspora_url(event.title, event_url(slug: event.slug)) }
       %i{:class => "fa-diapora"}
```
Comment (manno) [inline app/views/frontend/shared/_embedshare.haml]: Even though the item doesn't exist, maybe we should fix this typo too?

**Comment by paskalito** [comment]
ok theres more - im quiet^^

**Comment by manno** [comment]
well anyways ;)

**Pull request closed without merging**